Jenkinsfile

```diff
@@ -281,42 +285,36 @@ try {
             builds['rpm'] = {
```

Comment:
Can we do the rpm build for non-collaborators now (upon collaborator approval), or was there a reason besides the commit signature check to not?

Reply:
seems like we can..I'll make the change.